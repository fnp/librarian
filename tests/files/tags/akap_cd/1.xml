<akap_cd> dalszy-ciÄ…g-akapitu </akap_cd>
