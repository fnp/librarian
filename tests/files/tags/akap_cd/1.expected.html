<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">dalszy-ciÄ…g-akapitu</p>
</div>
